content/kubermatic/main/tutorials-howtos/cluster-rbac/_index.en.md

```diff
@@ -0,0 +1,108 @@
++++
+title = "Cluster RBAC"
+date = 2023-01-20T14:37:15+02:00
+weight = 14
+
++++
+
+This manual explains how to configure Role-Based Access Control (a.k.a RBAC) on user clusters.
+
+# Concepts
+You can grant permission to 3 types of subjects:
+* `user`: end-user identify by it email
+* `group`: named collection of users
+* `service account`: a k8s service account that authenticated a process (e.g. Continuous integration process)
```

Comment:
```suggestion
* `service account`: a Kubernetes service account that authenticates a process (e.g. Continuous integration)
```

Reply:
Let's spell out Kubernetes here and remove the duplicated "process" (it's already mentioned before the parentheses, I think that makes it clear already).